CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Improved the consistency of cross-compilation assistance provided across all supported `target_triple` and host OS/architecture combinations. [#769](https://github.com/heroku/libcnb.rs/pull/769)
+- Added cross-compilation assistance for `aarch64-unknown-linux-musl` (on macOS and ARM64 Linux) and `x86_64-unknown-linux-musl` (on ARM64 Linux). [#769](https://github.com/heroku/libcnb.rs/pull/769)
+
```

Comment:
```suggestion
```

Reply:
Lets pretend that never happened ;-) https://github.com/heroku/libcnb.rs/compare/d71b6c5c53cdea63953379969b87a2872bc0af77..f9fb1cfcc4030551941c860ad55004b44d90085b